claat/proto/constructor/helpers.go

```diff
@@ -140,6 +140,13 @@ func NewInlineContentTextStrongAndEmphasized(txt string) *tutorial.InlineContent
 	}
 }
 
+func NewHeading(level int32, contentSlice ...*tutorial.InlineContent) *tutorial.Heading {
```

Comment:
There seems to be a weird side effect of all the merges I've done that shows this as duplicate. Haven't been able to fix it, but this doesn't really add new code.